packages/discovery-provider/src/queries/get_attestation.py

```diff
@@ -162,10 +166,14 @@ def get_attestation(
         raise AttestationError(CHALLENGE_INCOMPLETE)
     if disbursement:
         raise AttestationError(ALREADY_DISBURSED)
-
+    now_utc = datetime.now(pytz.UTC)
+    if challenge_id in COOLDOWN_CHALLENGE_IDS and challenge.cooldown_days:
```

Comment:
Do we still need `COOLDOWN_CHALLENGE_IDS` if `cooldown_days` can tell us the same thing?